packages/eslint-mdx/src/parser.ts

```diff
@@ -189,7 +222,13 @@ export class Parser {
       traverse(root, {
         code,
         enter: (node, parent) => {
-          if (!ES_NODE_TYPES.includes(node.type)) {
+          if (
+            !is<Export | Import | Jsx>(
+              node,
+              (node: Node): node is Export | Import | Jsx =>
```

Comment:
Should not `Node` be the inferred type automatically?
```suggestion
              node: node is Export | Import | Jsx =>
```